config/ucx.conf

```diff
@@ -2,6 +2,8 @@
 CPU model=Grace
 UCX_REG_NONBLOCK_MEM_TYPES=host
 UCX_IB_MLX5_DEVX_OBJECTS=
+UCX_GDR_COPY_BW=0MBs,get_dedicated:30GBs,put_dedicated:30GBs
```

Comment:
Is the 30GBs from actual measurements?

Reply:
Yes, I got results close to 30GBs by measuring `copybw` benchmark with sizes > 1MB.